dev/service-registry/service-locator/src/main/scala/com/lightbend/lagom/discovery/ServiceRegistryActor.scala

```diff
@@ -39,6 +39,10 @@ class ServiceRegistryActor @Inject() (unmanagedServices: UnmanagedServices) exte
   private var router = PartialFunction.empty[Route, InetSocketAddress]
   private var routerFunctions = Seq.empty[PartialFunction[Route, InetSocketAddress]]
 
+  override def preStart(): Unit = {
+    rebuildRouter()
+  }
```

Comment:
This was an interesting bug - if no services registered themselves, then the unmanaged services never got added to the router.